cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -319,15 +391,29 @@ func generateMountsModeRun() error {
 		}
 	}
 
+	// 1.1a measure model
+	err := stampedAction("boot-model-measure", func() error {
+		return measureWhenPossible(func(tpm *secboot.TPMConnection) error {
+			model, err := loadModelFrom(filepath.Join(boot.InitramfsUbuntuBootDir,
+				"model"))
+			if err != nil {
+				return fmt.Errorf("cannot load run mode model: %v", err)
+			}
+			return secMeasureModel(tpm, model)
+		})
+	})
+	if err != nil {
+		return err
+	}
+
```

Comment:
we should add a TODO:UC20 abot x-checking the model after data is mounted

Reply:
added